fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -623,6 +653,9 @@ public void readFields(DataInput in) throws IOException {
             if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_62) {
                 exchangeInstanceParallel = in.readInt();
             }
+            if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_90) {
```

Comment:
this is unnecessary, we use json for  serialization now, this functiuon is deprecated